staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -433,6 +433,21 @@ func (s unstructuredJSONScheme) decodeToList(data []byte, list *UnstructuredList
 	return nil
 }
 
+type JSONFallbackEncoder struct {
```

Comment:
@liggitt @deads2k ptal where I use this to wrap the (now more strict) versioning codec.